config/ice_and_fire.cfg

```diff
@@ -189,13 +189,13 @@ all {
     B:"Generate All Structures on Superflat"=true
 
     # 1 out of this number chance per chunk for generation [range: 1 ~ 10000, default: 180]
-    I:"Generate Dragon Cave Chance"=180
```

Comment:
Decrease. Now that Lord has all the eggs he wants, the point is to reduce the amount of dragons overall.

Reply:
Ah, in that case I think this number should be *increased* rather than *decreased*. According to the documentation, `1 of N` chunks will generate stuff - by decreasing `N`, we increase the odds of spawning them. This is why we measure probabilities as a range `0% to 100%`, mod developers! >.<

Reply:
Ah, good point. They did it backwards. Stupid. The rest of this config will likely be in the same format, which also had the same changes applied in a similar manner.